### ISSUE TITLE
Cloudburst Totem - Fetch Tooltip Information stops showing if value is over 0

### ISSUE THREAD

**Issue by BenjaminCrooks** [WeakAuras/WeakAuras2#4528]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

Using %1.tooltip1 and %1.tooltip2 in a text sub element only shows the tooltip value when the value of tooltip2 (the amount of healing stored) is equal to 0; anything higher and the text no longer appears.

Have not tested with many other auras, but the ones that I did test worked as normal (Earth Shield and Water Shield).

Edit: After further testing the text occasional does appear, but only briefly. In addition, sometimes the trigger itself fails to recognize that the player has the aura active; so I do not think the issue is purely with the Fetch Tooltip Information option, and is probably with the trigger as a whole.

### WeakAuras Version

WeakAuras 5.6.0

### World of Warcraft Flavor

Retail (Default)

### World of Warcraft Region

US/NA

### Tested with only WeakAuras

- [X] Yes

### Reproduction Steps

1. Cast Cloudburst Totem
2. Heal

### Last Good Version

WeakAuras 5.5.0

### Export String

```Text
!WA:2!9rvWUTTrq0iOeKwAGIy1w3MIEWiOTa(IHTATBrVjQqf7cfjxs664CXAj5sYTMA3T7UuwY9ueqrZz9jOZ9K(e0xaHqr(aYH(b4VGo7skN0uGOduZm7SZ8M3m7uRtJHnIAe9YVAojKr9y5Iq8xENvOCvkt0NRimQ0smPFCSeRQTardb7NWiuvqBNE(oURczSSi2vuVRiC8GLRvDIsWP26GoyQsqssWc5h9nIkXxzRMWXUOCbQ5n5sSZyuOYJJZYooAWYySkm1hcKIWh4IhHHSDegLPsFTmpWOFIahtg76DIt3Ul1HrQVmjs(fb7FW3FWEFNDoLOc4zOjyXuzzKHkbqwFA2Kb3giV8yiqlUODlp)l88B56la80dneN6sHVsR5r4aWjFaWIJC6EsNt7olNwviw8nbKtgH9l1Fklc)x35gGFWckk7xGQgOW)4MlXyElagHkxeqQPZK4SydpwCpibsKcQbfoDoGlxCIM2)XTk2yfOgGcVmrWYPrV8Z4nu4XQlKPiGJFwvxH)qJrZNyMyicCOsWMYO4z3E8IVE)DvL8At(dERq1MLXe)uTA1Q)6sRRrNDRt97Z)yJryOGbXKeEgjsLA3cuVPmPKX4iJXxSQCc58kOT0C(VMlvK4jqLkykOk9bJ2963ZPydbuGAFMBCm0aJ6WVzvvdvXFqhb56T)5CueWqOT99)pKW6mXF0)JeUTw)2klf7SYC(vmr0zceF6zvcLWCerscYWdQyM3AwNV5X9654EHDFF)(p194NCK)7lF7FB(EFE18DqLUy9ixJFyfhUU9(MtnJG9p1V7X9C(NvjzSR6iW)woMgo5K7U3UnpOKp1hSegIFswvJnDQ2K(2RcYvkgT)iSaEzmxBUlMMOs)0fA51K5SK1x10mwQvFSEafMlRB8Cn4S1kPg)9crz46gy5NscVKILY6Mt6sa5nnzZgyCSiniftssvVWoJHI6ylHYEl3H5aTyzRFLSvX5wUHziPulfOGitvgrPzfvaziNjuZfMNk6kR4d4nGAUD16h4Di8qdpy669rd4FYAr943Jjse0RJsDVYm3AFnJnSMyCvznxXchv(29ZR9NF4JirlQ2hTTDEsZCq)q1r(c)F45NN)8fXcytHNspFw)MYUxhTjnWc8A7640Z9nZ2bWsXysILGq15aw0GY4PO6c9Me4H)HUsfsOSSjWkmRGy4pzQ1C4wrKYfYli0YzhqZAgIskfpSq0PyJIDwIrsnCmD27VkQQZDHENlKpDGKt1(yiUDShIiuZflgF)Ijfxx87GO6DT0q6EWUhU7Eng93p7Fd
```

### Export String 2 (only aura trigger)

```Text
!WA:2!Dj1ZUnntqCIQqfCLqIiu1NQ4acH4ahQ6xfnW16GJAlPTrRniOxAw7DS3v1zxRDxNM0Bnsi(o3hHC(7uEeYtGveIhaEeYtaZ6KYb8fpZS7o)(Zmn60Cqtwt2)9QPIeLmuvQtGN)Gf0slxPpVWkusJNE85PPgW2ygvMG17PesBC7GZIcilsuQCM6Az41IcO)87tdyza3310(tSArwgOnp5161H)03oUai0snD)LLgiyenXgwa55hZ6ppfSj8iSrwrbNadbeTJaAULtK0bGX7xMY46YHLPPIrZUS9HHrxggDij6ph1td4rKWEbD7oXSQ1gp)sPWgxKthdAnsZZL5J7p3XJ6RiyMDI)3dE3b79wnsRZq04tzqmctesy9rbD715tDVRuUwiEfpfzUyieTk)ufd()hSe9hqlP5FgvnAHFB5vauCicrILqrtLFNbYtR9XQhIayOwKcwemK(eiZz77SD1wlW0yAYvzAvPKfZbrg3ERFUIY64Be3aBtguI2KNVR3Bx9gpsso1yCrXwAo6d1HM6bBSyqHsBNQRbWjOQhv0e1z71dnK9i9G(tUFk2V4z3hgbJSFqyOX5aJtUwWS8B9VrPg0qpA96XuRkz4kf)pn((JFPGnB9u8f(Lz7xI5TShfPJE)fFT8IzPA0EdTOUPBSC1MvhxjhXIdBtccoJKOYv6t2a)IXvPurMNwiDyaCcnVGt3q78F0UArmwQ265lWjSxCk(ZW9MIVIjwTgptitv6bo)x6Dhvkwf2QQDNQT8LkjmhOghJazMLV5cMBMG34s3YkcPRxMjU7u7Dh7pGkK4BRoU6KnR(yv3QtXWG)Ustd5GDBT7Enh(JV87
```

**Comment by mrbuds** [comment]
It works for me when i change %1.tooltip3 in your aura to %1.tooltip2
What is your client language?

**Comment by mrbuds** [comment]
Also, have you really tested with only WeakAuras enable, or did you tick the checkbox because you had to to be able to submit the ticket?

**Comment by BenjaminCrooks** [comment]
Yes, I have tested this with with only WeakAuras enabled; every other addon was disabled when testing this. I realize my mistake in the export string makes it appear that I did not thoroughly and properly test this before submitting, so I apologize about that. Changing to %1.tooltip2 does not change the situation/fix the issue for me; I have fixed the export string to be the correct tooltip value #2. My client language is EN.

I also added an export string that is purely the Cloudburst Totem aura, with no tooltip info or any sub-elements; the issue persists for this WeakAura as well, where any amount of healing stored in the Cloudburst Totem no longer has the trigger returning as true, despite the aura still being the same.
